Gemfile

```diff
@@ -175,3 +175,5 @@ end
 
 extra_gems = File.expand_path("../Gemfile.local", __FILE__)
 eval File.read(extra_gems) if File.exist?(extra_gems)
+
+gem "omniauth-saml", "~> 1.10"
```

Comment:
Is including `omniauth-saml` intentional?  It feels outside the scope of this PR to me.

Reply:
I didn't change anything relating to circleCI. I did rename the image to 7.2.2 something; I guess we either need to update the image name circleci is using, or  build a new image using the same name.

Do we really want the version number hard coded in the circleci image name?